requirements.txt

```diff
@@ -44,3 +44,9 @@ questionary==1.10.0
 prompt-toolkit==3.0.28
 # Extensions to datetime library
 python-dateutil==2.8.2
+
```

Comment:
I'm trying to understand the reason for having these here, if wm install them from the one in execution repo